ibm/mas_devops/roles/suite_app_configure/vars/health.yml

```diff
@@ -3,6 +3,11 @@
 mas_app_ws_fqn: healthworkspaces.apps.mas.ibm.com
 mas_app_ws_apiversion: apps.mas.ibm.com/v1
 mas_app_ws_kind: HealthWorkspace
-
-mas_app_cfg_delay: 480
 mas_app_cfg_retries: 50
+mas_app_cfg_delay: 480
+mas_app_ws_spec:
```

Comment:
Bad merge?  Default specs are defined in a seperate file ... see vars/defaultspecs/xxx.yaml.